.github/workflows/test-marvin-nodejs.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Run tests
         run: npm test
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
@rwoll It seems this doesn't actually upload anything as it doesn't find `playwright-report`. This was already the case with v3 too from what I can tell. Should I remove it instead of upgrading it to v4?

Reply:
Ah!